### PR TITLE
New botany trait: Richer juice - doubles chem result

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
* Adds a new botany trait: Richer Juice
Doubles 2x chem results.
blocks Densified Chemicals. You can't take both.
available from 3D orange

## Why It's Good For The Game
new thing + nerf death nettle

This is primarily to nerf death nettle acid.
Basically, botany-spam plants shouldn't do much things in a single throw.
That's why we accepted the old botany nerf (flu-acid 50% to 25% => maximum 26u)

Now, flu-acid effect is so below:
26u of flu-acid is enough to remove `Captain's hat + PDA + jumpsuit`
13u of flu-acid is enough to remove `Jumpsuit` only

This means they wouldn't get the wanted result very quickly because they made a successful throw of a plant.

If botanist wants to get the original power level of acid, they should use this new trait.
but the new trait also blocks Densified Chemicals, which means the chem maximum will be limited to 50u.

### Wouldn't 2x chems be too powerful to other chems?
Every chem in botany is already too much. Toxins are like 8u, 20u, or something more.
but also toxins have very low metabolisation rate. Toxins will do its thing already with only 1u.

With consideration of this, having 2x means the example units become 16u, 40u, but it wouldn't make any big difference because it does the thing already.

and if they want to make very poisonous plant, they should use multiple toxins, not using this.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: Added a new botany trait: Richer Juice. Doubles 2x chem results. Not compatible with Densified Chemicals. You can't take both. available from 3D orange
balance: deathnettle now has 12% Fluorosulfuric acid
/:cl:

### CODE REVIEW

**Comment by BriggsIDP** [comment]
Was 25% > 12% picked for a specific reason?

**Comment by EvilDragonfiend** [comment]
> Was 25% > 12% picked for a specific reason?

It's just a damn chem, easily abusable.

File: code/modules/hydroponics/grown/nettle.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 	yield = 2
 	genes = list(/datum/plant_gene/trait/repeated_harvest, /datum/plant_gene/trait/plant_type/weed_hardy, /datum/plant_gene/trait/stinging)
 	mutatelist = list()
-	reagents_add = list(/datum/reagent/toxin/acid/fluacid = 0.25)
+	reagents_add = list(/datum/reagent/toxin/acid/fluacid = 0.12)
```
Comment (Rukofamicom) [inline code/modules/hydroponics/grown/nettle.dm:28]: A completely unrelated balance change doesn't belong in a feature PR.

**Review by Rukofamicom — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/nettle.dm (hunk 1 above)
Comment (Rukofamicom) [inline code/modules/hydroponics/grown/nettle.dm:28]: Completely unrelated balance change has no business being part of a feature PR. 
Should be atomized

File: code/modules/hydroponics/plant_genes.dm [hunk 2]
```diff
@@ -393,6 +393,7 @@
 	// 2x to max reagents volume.
 	name = "Densified Chemicals"
 	rate = 2
+	trait_id = "chem_boots"
```
Comment (Rukofamicom) [inline code/modules/hydroponics/plant_genes.dm]:
```suggestion
	trait_id = "chem_boost"
```

I feel like this is a typo.

File: code/modules/hydroponics/plant_genes.dm [hunk 3]
```diff
@@ -521,6 +522,12 @@
 			HY.update_icon()
 			HY.visible_message("<span class='warning'>The [H.myseed.plantname] spreads!</span>")
 
+// It boosts chemical output of a plant by rate
+/datum/plant_gene/trait/richer_juice
+	name = "Richer Juice"
+	rate = 2
+	trait_id = "chem_boots"
```
Comment (Rukofamicom) [inline code/modules/hydroponics/plant_genes.dm]:
```suggestion
	trait_id = "chem_boost"
```

Again, this just seems like a typo - just highlighting the other for easy access.

**Review by Rukofamicom — CHANGES_REQUESTED** [review]

**Comment by Rukofamicom** [comment]
Are you willing to expand on why you think the new trait is good for the game when botany is already so powerful as-is?

File: code/modules/hydroponics/grown/nettle.dm (hunk 1 above)
Comment (EvilDragonfiend) [reply code/modules/hydroponics/grown/nettle.dm:28]: the feature without rebalancing works like a revert of the previous nerf to the acid because it was 50% before, and adding this trait to nettle will make it back to 50%.
Now with this nerf, it's 13u at maximum, and 26u with the trait.
Chems being richer is fine anyway because most of the existing botany chems are all already too much, while a botanist shouldn't throw a single plant with a lot of acid in a short period.
* 13u is the possible maximum without the trait.
* 12u is enough to remove your jumpsuit.
* 26u is enough to remove your captain's hat, and PDA (including jumpsuit)
* 52u(if not untouched) will be enough to remove other stuff as well


I mean, a small acid with a single throw is fine, but it shouldn't be too much.
If they want to make it back to the same level of before-nerf, they should put the trait, but it will also block the maximum volume of a reagent.

**Comment by EvilDragonfiend** [comment]
> > new thing + nerf death nettle
> 
> Are you willing to expand on why you think the new trait is good for the game when botany is already so powerful as-is? This description is not really an explanation of anything.

answered in the review section.

**Comment by Rukofamicom** [comment]
> answered in the review section.

You only answered why death nettle was being nerfed. Why do you think adding a new trait which allows botany to double their output is good for the game when botany is already as powerful as it is?

You've addressed that you don't think it should be possible to double death nettle's output, but what about all the other extremely powerful things botany can produce? This is a pretty extreme increase in potential for all aspects of botany.

**Comment by EvilDragonfiend** [comment]
> You only answered why death nettle was being nerfed. Why do you think adding a new trait which allows botany to double their output is good for the game when botany is already as powerful as it is?
> You've included a nerf to death nettle so that its previous maximum is only possible with this trait, but what about all the other powerful things botany can produce? This is a pretty extreme increase in potential for all aspects of botany, even outside of direct combat.

I said most chems are already too much so it doesn't really do much even if your chems become double.

> Chems being richer is fine anyway because most of the existing botany chems are all already too much, while a botanist shouldn't throw a single plant with a lot of acid in a short period.

For example, toxins are usually like 8u, 20u, smh, and those units become 16u, 40u with this trait does really not make any difference because these toxins have usually very low metabolisation rate, and even small dose like 1u is enough to make them fucked up.
If they want to make a very lethal poisonous plant, they should use multiple toxins, not using this trait.
because this trait will lock the maximum volume to 50u (because you can't take densified chemicals)
but if you really want a powergaming poisonous plant, you'd want to have 100u with a lot of toxins.

**Comment by Rukofamicom** [comment]
So is this trait being introduced primarily for the sake of nerfing the initial death nettle, making it take extra steps to go back to its old power level?

**Comment by EvilDragonfiend** [comment]
> So is this trait being introduced primarily for the sake of nerfing the initial strength of death nettle, making it take extra steps to go back to its old power level?

primarily, yes, but also the old power level with damn mixed toxins will be limited.

**Comment by Rukofamicom** [comment]
Then the "why it's good for the game" section should summarize that instead of 

> new thing + nerf death nettle

**Review by Rukofamicom — APPROVED** [review]

**Comment by EvilDragonfiend** [comment]
> Then the "why it's good for the game" section should summarize that instead of
> 
> > new thing + nerf death nettle
> 
> I trust your experience with botany in any case, so I'm satisfied with this explanation now. I just don't think it was presented correctly.

changed!

**Review by PowerfulBacon — APPROVED** [review]
